client/navigation/NavigationStore.js

```diff
@@ -0,0 +1,31 @@
+
+/**
+ * Module dependencies.
+ */
+
+var createStore = require('dispatchr/utils/createStore');
+
+module.exports = createStore({
+    initialize: function () {
+    	this.state = {
+            currentPage: null
+        };
+    },
+    storeName: 'NavigationStore',
+    handlers: {
+        'NAVIGATE': function (route) {
+            var store = this.dispatcher.getStore('NavigationStore');
+            store.state.currentPage = route.name;
+            store.emit('NAVIGATE');
```

Comment:
When I implemented this previously, I had the event names as enum types, but it makes it kind of annoying to assign handlers, for example. We could change event names to lower case.